### PR TITLE
Added follower system in backend

### PR DESCRIPTION
**Pull request by CJVisser**

### CODE REVIEW

File: Project-Starlord/Controllers/FollowerController.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            _context = context;
+        }
+
+        [Authorize]
+        [HttpPost]
+        public async Task<ActionResult<FollowerModel>> PostFollowerModel(FollowerModel follower)
+        {
+            _context.Followers.Add(follower);
+            await _context.SaveChangesAsync();
```
Comment (MrTheGood) [inline Project-Starlord/Controllers/FollowerController.cs:30]: I'm curious, what is the reason why this is async?

File: Project-Starlord/Controllers/FollowerController.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            if (followerModel == null)
+            {
+                return NotFound();
+            }
+
+            _context.Followers.Remove(followerModel);
+            await _context.SaveChangesAsync();
+
+            return followerModel;
+        }
```
Comment (MrTheGood) [inline Project-Starlord/Controllers/FollowerController.cs:49]: So you can basically delete any follower if you have the id? Can't that cause any problems?

**Review by MrTheGood — APPROVED** [review]
Looks good, but I have some questions

File: Project-Starlord/Controllers/FollowerController.cs (hunk 1 above)
Comment (CJVisser) [reply Project-Starlord/Controllers/FollowerController.cs:30]: Short answer performance. For a good explanation read this: https://stackoverflow.com/a/30043327

File: Project-Starlord/Controllers/FollowerController.cs (hunk 2 above)
Comment (CJVisser) [reply Project-Starlord/Controllers/FollowerController.cs:49]: For now it will not be a problem. If it would be bigger then yes.

**Pull request merged**